src/main/java/com/android/volley/toolbox/HurlStack.java

```diff
@@ -153,7 +155,7 @@ private static boolean hasResponseBody(int requestMethod, int responseCode) {
      * Wrapper for a {@link HttpURLConnection}'s InputStream which disconnects the connection on
      * stream close.
      */
-    static class UrlConnectionInputStream extends FilterInputStream {
+    private static class UrlConnectionInputStream extends FilterInputStream {
```

Comment:
I think you can just leave this unchanged. I'm not sure if anything is using it but it shouldn't impact this PR. (I was concerned about *increasing* this classes visibility to protected, but it's fine to leave as is).